inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/utils/AgentUtils.java

```diff
@@ -344,8 +344,19 @@ public static String fetchLocalIp() {
      * check agent uuid from manager
      */
     public static String  fetchLocalUuid() {
-        String result = ExcuteLinux.exeCmd("dmidecode | grep UUID");
-        return AgentConfiguration.getAgentConf().get(AGENT_LOCAL_UUID, result);
+        String uuid = null;
+        try {
+            String result = ExcuteLinux.exeCmd("dmidecode | grep UUID");
+            if (!StringUtils.isEmpty(result) && StringUtils.containsIgnoreCase(result,"UUID")) {
+                String uuidResult = result.split(":")[1].trim();
+                 uuid = AgentConfiguration.getAgentConf().get(AGENT_LOCAL_UUID, uuidResult);
+                return uuid;
+            }
+        } catch (Exception e) {
+            LOGGER.error("fetch uuid  error");
+            e.printStackTrace();
+        }
+        return uuid;
```

Comment:
If agent conf has configured uuid,  your method still return null

Reply:
done

Reply:
Unless the result is null ,the uuid still return null; but whatever the agent conf has configured uuid ,the  "ExcuteLinux" execute all time ,still optimize